img/y/yelp.com.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 200 225"><path fill="#FF1A1A" d="m61.6 141.7 10.1-2.3 1-.3a11.2 11.2 0 0 0 8-13.3v-.2a11.2 11.2 0 0 0-1.8-3.8 14.1 14.1 0 0 0-4.1-3.5 38.3 38.3 0 0 0-5.9-2.8l-11.1-4a1048 1048 0 0 0-18.8-6.8c-4.1-1.5-7.6-2.8-10.6-3.7l-1.7-.5a29.5 29.5 0 0 0-8.3-1.6 10 10 0 0 0-4.3.8 10.8 10.8 0 0 0-3.6 2.6L9 104a21.5 21.5 0 0 0-1.1 2 17 17 0 0 0-1 2.2 57.8 57.8 0 0 0-3 19c0 5.8.2 13.3 3.4 18.4a10.8 10.8 0 0 0 3 3.3 13 13 0 0 0 7 2c3.6.2 7-.7 10.5-1.5l33.7-7.8Zm113.2-53.9a57.7 57.7 0 0 0-11-15.7 16.9 16.9 0 0 0-1.9-1.6l-1.9-1.3a21.6 21.6 0 0 0-2-1 10.7 10.7 0 0 0-4.5-.7 10 10 0 0 0-4.1 1.2c-2 1-4 2.5-6.9 5l-1.3 1.3c-2.3 2.1-4.8 4.8-7.9 7.9a1056.9 1056.9 0 0 0-13.9 14.4l-8.2 8.5a38.4 38.4 0 0 0-4.1 5 14.2 14.2 0 0 0-2.2 5 11.2 11.2 0 0 0 .1 4.2v.2a11.2 11.2 0 0 0 13 8.5l1-.2 44-10.2c3.4-.8 7-1.5 10-3.3 2.1-1.2 4.1-2.4 5.5-4.8a10.8 10.8 0 0 0 1.3-4.3c.6-6-2.5-12.8-5-18Zm-78.5 18.5c3.1-4 3.1-10 3.4-14.8 1-16.3 2-32.6 2.8-48.9.3-6.1 1-12.2.6-18.4-.3-5.1-.4-11-3.6-15.2-5.7-7.4-17.9-6.8-26.2-5.7a79.5 79.5 0 0 0-7.6 1.5c-2.6.6-5 1.2-7.5 2-8 2.6-19.2 7.4-21 16.6-1.2 5.2 1.4 10.5 3.3 15.2 2.4 5.8 5.7 11 8.7 16.4C57 69.3 65 83.4 73 97.7c2.3 4.2 5 9.6 9.5 11.8l1 .3c2 .8 4.3 1 6.4.4h.4a11.1 11.1 0 0 0 5.2-3.2l.8-.7Zm-3.9 43.5a10.2 10.2 0 0 0-11.3-3.8 10.4 10.4 0 0 0-3.4 1.9 26.3 26.3 0 0 0-4.7 4.8l-1.1 1.5-7 9.7a1047.7 1047.7 0 0 0-12 16.4l-6.5 9.3-1 1.5a30 30 0 0 0-3.8 7.7 10 10 0 0 0-.5 4.3c.2 1.6.8 3 1.5 4.3a23.2 23.2 0 0 0 1.4 2 21.3 21.3 0 0 0 1.6 1.6 17 17 0 0 0 2 1.6 61.8 61.8 0 0 0 15.8 7.7 56 56 0 0 0 14.4 2.8 17.3 17.3 0 0 0 2.5 0l2.3-.4a22 22 0 0 0 2.3-.7 11 11 0 0 0 3.8-2.4 10 10 0 0 0 2.3-3.7c.9-2 1.4-4.6 1.7-8.4l.2-1.8c.3-3.2.4-7 .6-11.3l.9-20.3.4-12c.1-2.7 0-5.8-.7-8.5a12.8 12.8 0 0 0-1.7-3.8Zm79.7 18.8a30.3 30.3 0 0 0-6.8-5.2l-1.6-1c-2.7-1.6-6-3.3-9.9-5.4a1106.5 1106.5 0 0 0-17.8-9.6l-10.6-5.6-1.6-.8c-2-1-4.2-1.9-6.4-2.3a13.7 13.7 0 0 0-2.4-.2 10.5 10.5 0 0 0-1.5 0 10.2 10.2 0 0 0-8.5 8.4 13 13 0 0 0 .2 4c.5 2.9 1.8 5.7 3 8.1l5.7 10.6a1051.2 1051.2 0 0 0 9.6 17.8c2 3.9 3.8 7.2 5.5 10l1 1.5c2 3.3 3.5 5.3 5.1 6.8a10 10 0 0 0 3.7 2.3c1.5.5 3 .6 4.6.5a21.7 21.7 0 0 0 2.3-.4 22 22 0 0 0 2.2-.7 17 17 0 0 0 2.3-1 56 56 0 0 0 11.7-8.9 62 62 0 0 0 12-16.1 23 23 0 0 0 1-4.6 11 11 0 0 0-.5-4.5 10 10 0 0 0-2.3-3.7Z"/></svg>
```

Comment:
Here's an optimized version without padding:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 176 220.5"><path fill="#ff1a1a" d="m57.7 139.1 10.2-2.3 1-.3a11.2 11.2 0 0 0 8-13.4v-.1c-.4-1.4-1-2.7-1.8-3.8a14.1 14.1 0 0 0-4.2-3.6c-1.8-1-3.8-2-5.8-2.7l-11.2-4a1048 1048 0 0 0-18.8-6.9c-4-1.4-7.5-2.7-10.5-3.6l-1.7-.5a29.5 29.5 0 0 0-8.4-1.6 10 10 0 0 0-7.9 3.3 22.4 22.4 0 0 0-2.6 3.8 57.8 57.8 0 0 0-4 21.2c0 5.9.2 13.4 3.4 18.4.8 1.3 1.8 2.5 3 3.3a13 13 0 0 0 7 2c3.6.3 7-.6 10.5-1.4l33.8-7.8zM171 85.2a57.7 57.7 0 0 0-13-17.3 22 22 0 0 0-4-2.3 9.3 9.3 0 0 0-4.4-.7 10 10 0 0 0-4.1 1.1c-2 1-4 2.5-6.8 5.1l-1.3 1.2-8 8-13.8 14.3-8.3 8.6a38.4 38.4 0 0 0-4 5 13.4 13.4 0 0 0-2.3 5c-.2 1.4-.1 2.8.1 4.2v.2a11.2 11.2 0 0 0 14.1 8.2l44-10.1c3.4-.8 6.9-1.5 10-3.3 2-1.2 4-2.4 5.4-4.9a9.4 9.4 0 0 0 1.3-4.3c.7-6-2.4-12.7-5-18zm-78.6 18.5c3.2-4 3.2-10 3.5-14.9 1-16.2 2-32.5 2.7-48.8.3-6.1 1-12.2.6-18.4-.3-5.2-.3-11-3.6-15.3C90-1 77.8-.5 69.4.7a79.5 79.5 0 0 0-15 3.5c-8 2.6-19.2 7.4-21.1 16.6-1.1 5.2 1.4 10.5 3.4 15.2 2.4 5.8 5.6 11 8.6 16.4C53.1 66.6 61.1 80.8 69.1 95c2.4 4.3 5 9.6 9.6 11.8l1 .4c2 .8 4.2 1 6.4.4h.4c2-.6 3.8-1.7 5.2-3.2l.7-.7zm-3.8 43.5a10.2 10.2 0 0 0-14.7-2 26.3 26.3 0 0 0-4.8 5l-1 1.3-7.1 9.8-12 16.4c-2.5 3.6-4.7 6.6-6.4 9.3l-1 1.5a30 30 0 0 0-3.9 7.7 10 10 0 0 0-.4 4.3c.2 1.5.7 3 1.5 4.3a23.2 23.2 0 0 0 3 3.6 61.8 61.8 0 0 0 34.7 12 22 22 0 0 0 4.5-1 10 10 0 0 0 6.2-6.1c.8-2 1.3-4.6 1.7-8.4l.2-1.8.6-11.4.8-20.2.4-12c.2-2.8 0-5.8-.7-8.6-.3-1.3-.9-2.5-1.6-3.7zm79.7 18.8a30.3 30.3 0 0 0-6.9-5.2l-1.5-1-10-5.4a610 610 0 0 0-17.8-9.6l-10.5-5.6-1.7-.8c-2-1-4.2-2-6.4-2.3a13.7 13.7 0 0 0-3.8-.2 10.2 10.2 0 0 0-8.4 12.4c.5 2.9 1.8 5.6 3.1 8l5.7 10.7c3 6 6.3 11.9 9.5 17.8l5.5 9.9 1 1.5a32 32 0 0 0 5.2 6.9 10 10 0 0 0 8.2 2.8 21.7 21.7 0 0 0 4.5-1.2 56 56 0 0 0 14-9.9 62 62 0 0 0 12-16 23 23 0 0 0 1-4.6c.2-1.5 0-3-.4-4.5a10 10 0 0 0-2.3-3.7z"/></svg>
```

Reply:
Or this:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 176 220.3"><path fill="#ff1a1a" d="M57.7 139.1c4.2-1 8.6-1.8 12.7-3.1a11.3 11.3 0 0 0 5-16.4c-3.7-5-10.2-6.5-15.7-8.7-12.5-4.5-24.9-9.2-37.5-13.2-4.7-1.5-10.7-2.7-14.7 1.1-5 4.7-6 11.7-7 18-.7 7.2-.8 14.5.9 21.5 1.1 4.7 4.6 9.4 9.8 9.8 6.8 1 13.3-1.6 19.8-2.8l26.7-6.2zM171 85.2a53 53 0 0 0-13.4-17.7c-3.7-2.9-9.2-3.7-13.2-.9-6.8 4.6-12.1 11.1-18 16.8-6.9 7.2-13.9 14.3-20.7 21.6-3.2 3.5-5.9 8.2-4.4 13 1.5 6 8.3 9.6 14.1 7.8l48.2-11.2c5-1.5 10.8-4.2 12-9.7 1.5-6.9-2-13.7-4.6-19.7zm-78.6 18.5c4.3-6.1 3-14 3.8-21 1.1-18.8 2.3-37.6 3.1-56.5-.3-7 .5-14.8-4.2-20.5-6-6.5-15.9-6-24-5.2a69 69 0 0 0-31.7 11.1 14.9 14.9 0 0 0-5.5 17c3.9 12.3 11.4 23 17.3 34.5 7.1 12.6 14 25.3 21.3 37.8 3 5.3 9.6 8.8 15.4 6.1 1.8-.7 3.3-2 4.5-3.3zm-3.8 43.5a10.3 10.3 0 0 0-15-1.7c-4.6 3.9-7.7 9.2-11.3 14-7.3 10-14.7 19.8-21.4 30.1-2.7 4.1-5 9.4-2.6 14.2 3.1 6 9.7 9 15.6 11.7a53 53 0 0 0 22.6 4.8c4.8-.2 9.7-3 11-7.9 2.2-7.5 1.8-15.5 2.3-23.3.5-11 1-21.9 1.2-32.9-.1-3-.7-6.6-2.4-9zm79.7 18.8c-5-5-11.8-8-18-11.4-10.3-5.6-20.6-11.3-31.1-16.6-4.4-2.2-10.1-3.4-14.2 0-3.9 2.8-4.8 8.2-3.1 12.5 2 5.7 5.2 10.8 8 16 5.5 10.4 10.9 21 16.9 31 2.5 4.1 6.2 8.7 11.6 8.5 6.6-.2 12-4.6 17-8.6a56.4 56.4 0 0 0 14.6-18.6c1.7-4.2 1.7-9.4-1.7-12.8z"/></svg>
```
![](https://d.tempfiles.download/D62D490704F9B5/?p=yhjGfmiZ)